docs-2.0/nebula-dashboard-en/4.cluster-operator/1.overview.md

```diff
@@ -0,0 +1,39 @@
+# 集群总览
+
+本文主要介绍 Dashboard 的集群总览页面。
+
+## 概览
+
+![overview](../figs/ds-001.png)
+
+Dashboard 的集群总览页面分为五个部分：
+
+- 集群概况
+- 集群信息
+- 节点监控
+- 状态列表
+- 服务监控
+
+### 集群概况
+
+在集群概况中，显示了节点数量，Graphd、Storaged、Metad正在运行服务及异常服务数量。在本示例中，Storaged存在异常服务为 **1**，可以点击 **查看** 按钮，快速查看异常服务。
+
+### 集群信息
+
+在集群信息中，显示了 **集群名称**、**创建时间**、**创建用户**、**版本** 的信息。
+
+!!! note
+
+    这里的版本信息为用户安装的Nebula Graph版本，而非Dashboard版本。
```

Comment:
please double check the desc here, or is it a bug?
![image](https://user-images.githubusercontent.com/78209557/140280003-26d724d3-2a8c-4d7b-8b81-2e830260b1fd.png)

Reply:
this is a bug, please add a issue